x-pack/plugins/security_solution/cypress/integration/cases/privileges.spec.ts

```diff
@@ -18,182 +18,26 @@ import {
-      },
-    ],
-  },
-};
-
-const secAllUser: User = {
-  username: 'sec_all_user',
-  password: 'password',
-  roles: [secAll.name],
-};
-
-const secReadCasesAll: Role = {
-  name: 'sec_read_cases_all_role',
-  privileges: {
-    elasticsearch: {
-      indices: [
-        {
-          names: ['*'],
-          privileges: ['all'],
-        },
-      ],
-    },
-    kibana: [
-      {
-        feature: {
-          siem: ['minimal_read', 'cases_all'],
-          actions: ['all'],
-          actionsSimulators: ['all'],
-        },
-        spaces: ['*'],
-      },
-    ],
-  },
-};
-
-const secReadCasesAllUser: User = {
-  username: 'sec_read_cases_all_user',
-  password: 'password',
-  roles: [secReadCasesAll.name],
-};
-
 const usersToCreate = [secAllUser, secReadCasesAllUser];
 const rolesToCreate = [secAll, secReadCasesAll];
-
-const getUserInfo = (user: User): UserInfo => ({
-  username: user.username,
-  full_name: user.username.replace('_', ' '),
-  email: `${user.username}@elastic.co`,
-});
-
-const createUsersAndRoles = (users: User[], roles: Role[]) => {
-  const envUser = getEnvAuth();
-  for (const role of roles) {
-    cy.log(`Creating role: ${JSON.stringify(role)}`);
-    cy.request({
-      body: role.privileges,
-      headers: { 'kbn-xsrf': 'cypress-creds-via-config' },
-      method: 'PUT',
-      url: constructUrlWithUser(envUser, `/api/security/role/${role.name}`),
-    })
-      .its('status')
-      .should('eql', 204);
-  }
-
-  for (const user of users) {
-    const userInfo = getUserInfo(user);
-    cy.log(`Creating user: ${JSON.stringify(user)}`);
-    cy.request({
-      body: {
-        username: user.username,
-        password: user.password,
-        roles: user.roles,
-        full_name: userInfo.full_name,
-        email: userInfo.email,
-      },
-      headers: { 'kbn-xsrf': 'cypress-creds-via-config' },
-      method: 'POST',
-      url: constructUrlWithUser(envUser, `/internal/security/users/${user.username}`),
-    })
-      .its('status')
-      .should('eql', 200);
-  }
-};
-
-const deleteUsersAndRoles = (users: User[], roles: Role[]) => {
-  const envUser = getEnvAuth();
-  for (const user of users) {
-    cy.log(`Deleting user: ${JSON.stringify(user)}`);
-    cy.request({
-      headers: { 'kbn-xsrf': 'cypress-creds-via-config' },
-      method: 'DELETE',
-      url: constructUrlWithUser(envUser, `/internal/security/users/${user.username}`),
-      failOnStatusCode: false,
-    })
-      .its('status')
-      .should('oneOf', [204, 404]);
-  }
-
-  for (const role of roles) {
-    cy.log(`Deleting role: ${JSON.stringify(role)}`);
-    cy.request({
-      headers: { 'kbn-xsrf': 'cypress-creds-via-config' },
-      method: 'DELETE',
-      url: constructUrlWithUser(envUser, `/api/security/role/${role.name}`),
-      failOnStatusCode: false,
-    })
-      .its('status')
-      .should('oneOf', [204, 404]);
-  }
+// needed to generate index pattern
+const visitSecuritySolution = () => {
```

Comment:
Worth moving into a util? If you end up moving it to a util, should probably rename it to something like `loadSecuritySolutionIndexPatterns`